k8s/openebs-volumepolicies.yaml

```diff
@@ -228,11 +228,11 @@ metadata:
 spec:
   policies:
   - name: VolumeMonitor
-    value: openebs/m-exporter:0.5.0
```

Comment:
Let us remove anything related to volume policies from 0.5.3

Reply:
this merge is for master branch. should we retain it?